docs/src/lib/services/onboard.js

```diff
@@ -82,6 +84,7 @@ const intiOnboard = async () => {
       coinbase,
       ledger,
       trezor,
+      frontier,
```

Comment:
```suggestion
```

Reply:
This will either need to be removed or the Frontier `-alpha` package added to the [docs package.json](https://github.com/blocknative/web3-onboard/blob/v2-web3-onboard-develop/docs/package.json) - It would also be good to add a frontier `wallet` docs page similar to other wallets [here](https://github.com/blocknative/web3-onboard/tree/v2-web3-onboard-develop/docs/src/routes/docs/%5B...4%5Dwallets)